src/tests/system/tests/test_autofs.py

```diff
@@ -20,9 +20,11 @@
 @pytest.mark.parametrize("sssd_service_user", ("root", "sssd"))
 @pytest.mark.require(
     lambda client, sssd_service_user: ((sssd_service_user == "root") or client.features["non-privileged"]),
-    "SSSD was built without support for running under non-root"
+    "SSSD was built without support for running under non-root",
```

Comment:
I saw this in my PR, but failed to understand rationale...

Reply:
What sense does this trailing ',' make?

Reply:
I dont actually know but from black [docs](https://test-black.readthedocs.io/en/style-guide/style_guide/trailing_commas.html):

_Black will add trailing commas to expressions that are split by comma where each element is on its own line. This includes function signatures._

_One exception to adding trailing commas is function signatures containing *, *args, or **kwargs. In this case a trailing comma is only safe to use on Python 3.6. Black will detect if your file is already 3.6+ only and use trailing commas in this situation._

Reply:
And I also found [this](https://test-black.readthedocs.io/en/style-guide/the_black_code_style.html#how-black-wraps-lines):

_You might have noticed that closing brackets are always dedented and that a trailing comma is always added. Such formatting produces smaller diffs; when you add or remove an element, it’s always just one line. Also, having the closing bracket dedented provides a clear delimiter between two distinct sections of the code that otherwise share the same indentation level (like the arguments list and the docstring in the example above)._

Reply:
Ok, thanks.